sdk/communication/Azure.Communication.Messages/src/Generated/tspCodeModel.json

```diff
@@ -445,7 +445,7 @@
    "Accessibility": "internal",
    "Description": "A data stream.",
    "IsNullable": false,
-   "Usage": "Output",
+   "Usage": "None",
```

Comment:
It should be `Output`, but I think it's better to let it go, because:
- it doesn't impact the generated codes
- we know there are unknown issues in our own logic of calculating `Usage`. So I choose to ignore it for now, until we adopt TCGC `getAllModels`